AGENTS.md

```
@@ -258,6 +258,33 @@ For deeper implementation guidance, use `.claude/skills/entitlements.md`.
 
 ---
 
+## PR Discipline (Required)
+
+### Size Limits
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)